Document additional artifacts passing to TF (#655)
packit/packit.dev: content/docs/testing-farm.md

* **tmt_plan** - Run plans by the given name. Can be passed as a regular
  expression.
* **tf_post_install_script** - Bash script as a string to run during the guest provisioning. 

> * **tf_extra_params** - a free-form dict that allows specifying extra parameters to Testing Farm. 

Reviewer: ```suggestion
* **tf_extra_params** - a free-form dictionary that allows specifying extra parameters to the Testing Farm. 
```

> For a complete list of parameters, refer to [Testing Farm documentation](https://testing-farm.gitlab.io/api/#operation/requestsPost). 

Reviewer: ```suggestion
For a complete list of parameters, refer to the [Testing Farm documentation](https://testing-farm.gitlab.io/api/#operation/requestsPost). 
```

> The dict must follow the structure of the Testing Farm request. Options specified in the dict have the 

Reviewer: ```suggestion
The dictionary must follow the structure of the Testing Farm request. Options specified in the dictionary have the 
```

highest precedence, i.e. can override Packit's defaults. They are being merged with the Packit's values, the only exception
is the `artifacts` list, which is combined with the artifact passed by Packit. Also, beware of indentation-sensitivity of the YAML format. 

> You can verify that the option is processed correctly using a [YAML parser](https://yaml-online-parser.appspot.com/). Refer to [configuration examples](/docs/configuration/examples/#tests) for more information.

Reviewer: ```suggestion
You can verify that the option is processed correctly using a [YAML parser](https://yaml-online-parser.appspot.com/). Refer to the [configuration examples](/docs/configuration/examples/#tests) for more information.
```
